### ISSUE TITLE
objective function and eval_metric in XGBoost survival analysis

### ISSUE THREAD

**Issue by liuhongwei2018** [IyarLin/survXgboost#1]
Hi, 
Can you provide some documentation explaining how XGBoost makes changes to the survival analysis, such as what the loss function is and what the evaluation metrics is (cox-nloglik)?

**Comment by IyarLin** [comment]
Hi,

The XGBoost package does not contain much documentation for the Cox regression implementation. 

The evaluation metric and I believe also the loss function is the [negative partial Cox likelihood](https://en.wikipedia.org/wiki/Proportional_hazards_model#The_Cox_model). 

Hope this helps.

Iyar
